src/Nethermind/Nethermind.Evm/EOF/EvmObjectFormat.cs

```diff
@@ -9,6 +9,8 @@
 using Nethermind.Core.Extensions;
 using Nethermind.Logging;
 
+[assembly: InternalsVisibleTo("Nethermind.EofParser")]
+
 namespace Nethermind.Evm.EOF;
```

Comment:
Could you please change `EOF` to `Eof` to follow the .NET naming convention for 3+ letter shorthands? Both the namespace and directory.

Reply:
I think this review should be in this PR : https://github.com/NethermindEth/nethermind/pull/4950